content/docs/notebooks/setup.md

```diff
@@ -265,4 +265,6 @@ exposed to the internet and is an unsecured endpoint by default.
 * Learn the advanced features available from a Kubeflow notebook, such as
   [submitting Kubernetes resources](/docs/notebooks/submit-kubernetes/) or
   [building Docker images](/docs/notebooks/submit-docker-image/). 
-  
\ No newline at end of file
+* Visit the [troubleshooting guide](docs/notebooks/troubleshoot) for fixing common
```

Comment:
The link gives a 404. Please follow the pattern of the link above.

See preview:
https://deploy-preview-1008--competent-brattain-de2d6d.netlify.com/docs/notebooks/setup/#next-steps